### PR TITLE
Removing FireBase for Actions

### LINKED ISSUE #204: [ BUG ] Github Actions failing when we review or request changes in PR
image

## I think this are the runs when I reviewed PR or asked for changes. I don't know why this happens ? Please let me know the possible reason for this

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Fixes #204 

## Proposed Changes

Worked on the GitHub actions removing FireBase.
PS: Despite rebasing my repository , I see changes that have already been merged. Please excuse me for that @rohan-kulkarni-25 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/273"></a>

### CODE REVIEW

File: .github/workflows/community.yml
Comment (Panquesito7) [inline .github/workflows/community.yml:33]: What does this actually do? 🤔

Comment (AbhijithGanesh) [reply .github/workflows/community.yml:33]: It basically is a firebase connection for GitHub actions.( I suppose). Since we no longer use FireBase , it was suggested that we remove them.

File: public/data/AbhijithGanesh.json
Comment (Panquesito7) [inline public/data/AbhijithGanesh.json:22]: Was this added before in another PR, or it was added in this PR?

Comment (AbhijithGanesh) [reply public/data/AbhijithGanesh.json:22]: It was added in the previous PR and I merged it too. I don't know why it reflected and I didn't want to create another branch again. I rebased my repo before merging but it still shows the same changes.

Comment (AbhijithGanesh) [reply public/data/AbhijithGanesh.json:22]: I realize that my changes haven't reflected despite Eddie approving them, it will reflect once this merge is approved.

**Comment by rohan-kulkarni-25** [comment]
Hey @AbhijithGanesh  !! Thanks for creating PR but It seems that you created a PR from main branch that's why all these changes are reflected. I suggest to please create a branch and then create a PR that will help you to keep each change in seperate PR and don't affect each other. If possible Please go for it now because it will be easy for reviews and approvals

**Comment by AbhijithGanesh** [comment]
I agree with the branching approach to things except for this repo I had locally created a branch and merged it before I sent it here. I rebased my repository locally , merged then sent in a PR(for this case) . I shall create branches from the next time.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
It would be better if you re-create a PR for this, as the Git diff displays that you made other irrelevant changes.

**Pull request closed without merging**
